api/clans.py

```diff
@@ -0,0 +1,24 @@
+import faf.db as db
+
+from api import *
+
+@app.route('/clans')
+def clans_get():
+    clans = []
+    with db.connection.cursor(db.pymysql.cursors.DictCursor) as cursor:
+        cursor.execute('SELECT * FROM clans WHERE status = 1')
+        clans = cursor.fetchall()
+    return { 'data': clans }
```

Comment:
He means the API should be compliant to http://jsonapi.org/

There are various problems with this implementation
* It is not compliant to the JSONAPI structure
* It exposes the table structure directly to the API - this should be avoided since API and DB should be decoupled (a change in the DB should not be a change to the API)
* It does not support sorting, paging, limiting, selecting fields
* It is inconsistent with other API implementations

By using `query_commons.py` all these problems would've been avoided